src/test/java/com/googleinterns/zoomtube/servlets/AuthenticationServletTest.java

```diff
@@ -0,0 +1,140 @@
+package com.googleinterns.zoomtube.servlets;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.googleinterns.zoomtube.data.AuthenticationStatus;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+
+public class AuthenticationServletTest {
+  private AuthenticationServlet servlet;
+  private MockHttpServletRequest request;
+  private MockHttpServletResponse response;
+
+  @Before
+  public void setUp() throws ServletException {
+    servlet = new AuthenticationServlet();
+    servlet.init();
+    request = new MockHttpServletRequest();
+    response = new MockHttpServletResponse();
+  }
+
+  @Test
+  public void doGet_loggedIn_true() throws ServletException, IOException {
+    LocalServiceTestHelper helper =
```

Comment:
helper is not a very descriptive variable name. I recommend localService.

"Avoid filler names, like temp, var, data, val, helper, util, etc. that don't aid the reader's understanding."
See: https://docs.google.com/document/d/1_ldClEZLsI1cMpQl_BfeIlkjTsE2IiJ2PgvhABfO6vs/edit#

Reply:
Changed